### DISCUSSION TITLE
Add multiple button to repeater

### DISCUSSION THREAD

**Discussion by mstfkhazaal** [filamentphp/filament#8153] [Ideas]
Add multiple button to repeater

**Comment by leandrocfe** [comment]
What does it mean?

**Reply by mstfkhazaal** [reply]
Like this
![reorderable-with-buttons copy](https://github.com/filamentphp/filament/assets/47450090/9358078c-3e5d-43c0-91aa-bbe079c975fc)

**Comment by leandrocfe** [comment]
Filament v3 supports Actions inside the form. You can do this
```php
Actions::make([
    Action::make('add_random_member')
        ->color('gray')
        ->action(function () {
            // do something
        }),
    Action::make('add_5_members')
        ->color('gray')
        ->action(function () {
            // do something
        })
])
    ->alignEnd(),

Repeater::make('members')
...
```
![image](https://github.com/filamentphp/filament/assets/3833889/19b9fee0-81f6-4d61-9ccb-bdd4a54bc8d7)
